plugins/skip-button-plugin/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@srgssr/skip-button-plugin",
+  "version": "0.0.1",
+  "type": "module",
+  "main": "dist/skip-button-plugin.cjs.js",
+  "module": "dist/skip-button-plugin.es.js",
+  "style": "./dist/skip-button-plugin.min.css",
+  "exports": {
+    ".": {
+      "import": "./dist/skip-button-plugin.es.js",
+      "require": "./dist/skip-button-plugin.cjs.js"
+    }
+  },
+  "files": [
+    "dist/*",
+    "scss/*"
+  ],
+  "targets": {
+    "main": false,
+    "github-page": {
+      "publicUrl": "./",
+      "isLibrary": false,
+      "outputFormat": "esmodule"
+    }
+  },
+  "scripts": {
+    "build": "npm run build:lib && npm run build:css",
+    "build:css": "sass ./scss/skip-button-plugin.scss:dist/skip-button-plugin.min.css --style compressed --source-map --load-path node_modules",
+    "build:lib": "rollup -c",
+    "github:page": "parcel build index.html --target github-page",
+    "start": "parcel -p 4200 index.html --open",
```

Comment:
since you're using `vite` for everything else, why not get ride of `parcel` ?

Reply:
I tried to do it but I was unsuccessful there was already too much in my plate with the test framework migration and the plugin. I would rather do it in a future PR isolated from this change.